plugins/processors/k8sdecorator/stores/podstore_test.go

```diff
@@ -429,13 +429,12 @@ func (client *MockClient2) Shutdown() {
 func TestGetJobNamePrefix(t *testing.T) {
 	assert.Equal(t, "abcd", getJobNamePrefix("abcd-efg"))
 	assert.Equal(t, "abcd", getJobNamePrefix("abcd.efg"))
-	assert.Equal(t, "abcd", getJobNamePrefix("abcd-e.fg"))
-	assert.Equal(t, "abc", getJobNamePrefix("abc.d-efg"))
-	assert.Equal(t, "abcd", getJobNamePrefix("abcd-.efg"))
-	assert.Equal(t, "abcd", getJobNamePrefix("abcd.-efg"))
```

Comment:
Could you explain why those test cases are wrong?

Reply:
according to what I read, it's not possible to have . and - next to each other in DNS subdomain name, the regexp given by k8s when creating such a resource (in the error message) is: `[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*`.

So I made a mistake I think for the first two apparently, I will fix it.

Reply:
actually it's ok, I didn't delete the first two, only changed the expected string